buildSrc/build.gradle

```diff
@@ -7,6 +7,8 @@ repositories {
 }
 
 dependencies {
+    implementation "gradle.plugin.com.github.johnrengelman:shadow:7.1.2"
+    implementation "org.aim42:htmlSanityCheck:1.1.6"
     implementation "io.micronaut.build.internal:micronaut-gradle-plugins:5.2.3"
```

Comment:
should not this be 5.3.0?

Reply:
It wasn't released when I created the PR, will update :)